### PR TITLE
Angela M.'s responses rails-api-relationship-diagnostic.

### PR DESCRIPTION
**Pull request by angmas**
comfort: 5
clarity: 5

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -40,31 +62,44 @@ like to show all movies favorited by a profile on
...
 
   ```md
-    # < Your Response Here >
+  The serializer controls the format of the returned data. It can act as a filter.
+
   ```
 
   ```rb
   class ProfileSerializer < ActiveModel::Serializer
+    attributes :id, :user_name, :movies
```
Comment (benjimelito) [inline diagnostic.md:71]: Don't forget this line: `has_many :movies`

**Comment by benjimelito** [comment]
Very nice, 6/7

**Comment by angmas** [comment]
Hey Ben, shouldn't the "has_many" macro actually be in the Profile class
and not in the ProfileSerializer class?

class Profile < ApplicationRecord

has_many :movies, through: :favorites

has_many :favorites, dependent: :destroy

end

Thanks,
Angela

**Comment by benjimelito** [comment]
I may have been taking the solution branch too literally - if that way worked in class then have yourself another point! 7/7

**Pull request closed without merging**